lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -193,24 +199,25 @@ newTransactionLayer
     :: forall (n :: NetworkDiscriminant) k t.
         ( t ~ IO Shelley
         , WalletKey k
+        , Typeable n
         )
     => Proxy n
     -> ProtocolMagic
     -> EpochLength
     -> TransactionLayer t k
-newTransactionLayer _proxy _protocolMagic epochLength = TransactionLayer
-    { mkStdTx = \acc ks tip -> mkTx emptyTxPayload (defaultTTL epochLength tip) acc ks
+newTransactionLayer proxy _protocolMagic epochLength = TransactionLayer
```

Comment:
in the light of this change - we need still protocolMagic, right?

Reply:
Actually no :thinking: ... Since there's only one testnet in Shelley, the network is either mainnet or testnet.

Reply:
hehe, it was supposed to be a question - so we still need protocol magic (probably redundant - agree here, hence stopped at this place)

Reply:
on the other hand we will need protocol magic for icarus/byron witnesses very soon

Reply:
We do need it for the Byron transaction layer indeed, and for the associated witnesses :)